src/network/networkd-route.c

```diff
@@ -2121,6 +2121,7 @@ int config_parse_tcp_advmss(
 
         if (isempty(rvalue)) {
                 n->advmss = 0;
+                TAKE_PTR(n);
```

Comment:
No need to store zero values as it has no use.

Reply:
You mean, we should drop `if (isempty(rvalue))` block? That's also a good way. If you like it, I will drop the block.

Reply:
What you think ? Should we go ahead with rest values which  good or drop it altogether. Hmm pretty confusing

Reply:
Hm? The problem here is, if we do not have `TAKE_PTR()` in this block, then setting advmss with an empty string causes the [Route] section invalid. So, we need to do one of the following:
1. add `TAKE_PTR()` in the block, then we accept an empty string for the setting.
2. drop the `if` block to refuse an empty string.

I have no preference. So, if you like the second way, I will drop the block.

Reply:
Ok lets allow this and for future too.

Reply:
Thanks.